front/src/api/books.tsx

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+
+import { config } from "../config";
+
+export const books = {
+  fetchBooks: (pageNumber: number) => {
```

Comment:
ogólnie takie funkcje api powinny służyć temu, żeby zwrócić już przetworzone dane. Więc najlepiej, jakbys tutaj zwracal response.data, zeby potem dalej w kodzie juz sie w to nie wkopywac jakimś "data.data".

Reply:
Wiem ale gdy próbowałem je przetworzyć od razu to mi nie wychodziło i były jakieś dziwne błędy walczyłem z tym kilka godzin i odpuściłem.
During